doc/configure-the-bundle.md

```diff
@@ -18,6 +18,7 @@ docusign:
         private_key: "YourPrivateKey"
         integration_key: "YourIntegrationKey"
         user_guid: "YourUserId"
+        grant_type: authorization_code # Grant type (authorization_code or implicit)
```

Comment:
```suggestion
        response_type: "authorization_code" # Grant type (authorization_code or implicit)
```

Reply:
The response type is not the same thing than grant type.
* Grant type: the grant type selected in DocuSign (`authorization code grant type` vs `implicit grant type`)
* Response type: `code` vs `token` according to the grant type selected

If we set the response type here, the user would have to search in the DocuSign documentation which one to use according to its grant type. Whereas the user already knows which grant type he/she selected.

Reply:
ok I see, agreed for the configuration key but everywhere else it should be response_type

Reply:
Fixed